tests/testdata/codegen/transpiled_examples/azure-app-service-pp/python/azure-app-service.py

```diff
@@ -17,18 +17,24 @@
     resource_group_name=appservicegroup.name,
     account_name=sa.name,
     public_access=azure_native.storage.PublicAccess.NONE)
-blob_access_token = pulumi.Output.secret(pulumi.Output.all(sa.name, appservicegroup.name, sa.name, container.name).apply(lambda saName, appservicegroupName, saName1, containerName: azure_native.storage.list_storage_account_service_sas_output(account_name=sa_name,
```

Comment:
This apply wasn't correct, all returns a single list value, so this apply would fail that the lambda was called without it's other required positional arguments.